README.org

```diff
@@ -34,4 +34,5 @@ manual intervention, use:
   (add-hook 'help-mode-hook #'epithet-rename-buffer)
   (add-hook 'occur-mode-hook #'epithet-rename-buffer)
   (add-hook 'shell-mode-hook #'epithet-rename-buffer)
+  (add-hook 'compilation-start-hook #'epithet-rename-buffer)
```

Comment:
Does this work? Or do you need to use the `epithet-rename-buffer-ignoring-arguments` function you defined?

Reply:
obviously it doesn't :-D. Thank you for catching that. It is fixed now.